ui/app/components/control-group-success.js

```diff
@@ -6,11 +6,12 @@
 import { service } from '@ember/service';
 import Component from '@ember/component';
 import { task } from 'ember-concurrency';
+import apiErrorMessage from 'vault/utils/api-error-message';
 
 export default Component.extend({
```

Comment:
I started glimmerizing this component and converting it to `ts` but it lead to a multitude of Ember Data models that needed to be typed which just didn't seem worth the time.

Reply:
Smart rabbit hole jump!